tests/integration/jumps_test.py

```diff
@@ -1,39 +1,108 @@
 
 
 @pytest.vaspxml_available
-class TestSites:  # type: ignore
+class TestJumps:  # type: ignore
     n_parts = 10
     diffusing_element = 'Li'
 
-    def test_site_inner_fraction(self, vasp_traj, vasp_sites, structure):
-        transitions = Transitions.from_trajectory(trajectory=vasp_traj,
-                                                  structure=structure,
-                                                  floating_specie='Li',
-                                                  site_inner_fraction=0.5)
-        jumps = Jumps(transitions=transitions,
-                      sites=vasp_sites,
-                      minimal_residence=100)
+    def test_site_inner_fraction(self, vasp_traj, structure):
+        transitions = vasp_traj.transitions_between_sites(
+            sites=structure, floating_specie='Li', site_inner_fraction=0.5)
+        jumps = Jumps(transitions=transitions, minimal_residence=100)
 
         assert len(jumps.data) == 267
         assert np.all(jumps.data[::100].to_numpy() == np.array([[
             0, 94, 0, 282, 303
         ], [15, 74, 8, 1271, 1286], [34, 49, 33, 3141, 3296]]))
 
-    def test_minimal_residency(self, vasp_traj, vasp_sites, structure):
-        transitions = Transitions.from_trajectory(trajectory=vasp_traj,
-                                                  structure=structure,
-                                                  floating_specie='Li')
-        jumps = Jumps(transitions=transitions,
-                      sites=vasp_sites,
-                      minimal_residence=3)
+    def test_minimal_residency(self, vasp_traj, structure):
+        transitions = vasp_traj.transitions_between_sites(sites=structure,
+                                                          floating_specie='Li')
+        jumps = Jumps(transitions=transitions, minimal_residence=3)
 
         assert len(jumps.data) == 462
         assert np.all(jumps.data[::200].to_numpy() == np.array([[
             0, 94, 0, 282, 284
         ], [18, 54, 24, 2937, 3015], [41, 41, 67, 849, 851]]))
+
+    def test_n_jumps(self, vasp_jumps):
+        assert vasp_jumps.n_solo_jumps == 450
+        assert vasp_jumps.n_jumps == 462
+        assert isclose(vasp_jumps.solo_fraction, 0.974026, abs_tol=1e-4)
+
+    def test_rates(self, vasp_jumps):
+        rates = vasp_jumps.rates(n_parts=10)
+        assert isinstance(rates, pd.DataFrame)
+        assert len(rates) == 1
+
+        row = rates.loc[('48h', '48h')]
+
+        assert isclose(row['rates'], 1174999999999.9998)
+        assert isclose(row['std'], 90769080986.31458)
+
+    def test_activation_energies(self, vasp_jumps):
+        activation_energies = vasp_jumps.activation_energies(n_parts=10)
+
+        assert isinstance(activation_energies, pd.DataFrame)
+        assert len(activation_energies) == 1
+
+        row = activation_energies.loc[('48h', '48h')]
+
+        assert isclose(row['energy'], 0.134486, abs_tol=1e-4)
+        assert isclose(row['std'], 0.00405952, abs_tol=1e-6)
```

Comment:
The values have changed but I remember that you mentioned this and that it is now the old value again (so I leave a comment here, but I assume it is correct now)

Reply:
This was from a bug in #228, see https://github.com/GEMDAT-repos/GEMDAT/pull/253